mobile/src/components/uam-configure-group/common/types/selected-items-values.type.ts

```diff
@@ -0,0 +1,5 @@
+type SelectedItemsValues = {
+  [x: string]: boolean;
+};
```

Comment:
As alternative
```suggestion
type SelectedItemsValues = Record<string, boolean>;
```

Reply:
Yes, I think we can get rid of this type at all, we use it only in one place